src/app/composable/themeSwitch.ts

```diff
@@ -6,12 +6,14 @@ export const useThemeSwitcher = (key: string, initialValue: string) => {
     const storedValue: string | null = localStorage.getItem(key)
     if(storedValue !== null) {
         value.set(storedValue)
+        document.body.setAttribute('data-theme', storedValue);
     }
 
     function storageHandler(e: StorageEvent): void {
         if(e.key === key) {
             const newValue: string = e.newValue !== null ? e.newValue : ''
```

Comment:
refactor this to  const newValue: string = e.newValue ?? '';

Reply:
refactor done